pkg/channel/channel.go

```diff
@@ -78,3 +78,31 @@ func ListChannel(osnURL string, caCertPool *x509.CertPool, tlsClientCert tls.Cer
 
 	return channels, nil
 }
+
+func ListChannelOnPeer(id identity.SigningIdentity, connection pb.EndorserClient) ([]*pb.ChannelInfo, error) {
```

Comment:
All of the other functions that interact with network nodes using gRPC have signatures of the following form:
```suggestion
func ListChannelOnPeer(ctx context.Context, connection grpc.ClientConnInterface, id identity.SigningIdentity) ([]*pb.ChannelInfo, error) {
```

I suggest maintaining a consistent API structure.